Makefile

```diff
@@ -4,7 +4,7 @@ GOLANG_IMG = golang:1.11
 PROMBENCH_DIR = /go/src/github.com/prometheus/prombench
 USERID = $(shell id -u ${USER})
 USERGROUP = $(shell id -g ${USER})
-DOCKER_CMD = docker run --rm \
+DOCKER_CMD = docker run --rm\
```

Comment:
instead of `@$(DOCKER_CMD) chown ${USERID}:${USERGROUP} `
you should be able to do `@$(DOCKER_CMD) = docker run -u $USERID:$USERGROUP`

Reply:
I tried doing that earlier, but it gives me the following error:
```
go: disabling cache (/.cache/go-build) due to initialization failure: mkdir /.cache: permission denied
go: cannot use modules with build cache disabled
```

Reply:
ok go it , lets leave as is.

Reply:
or maybe try setting GOCACHE or HOME envs

https://github.com/golang/go/issues/26280

Reply:
https://golang.org/pkg/cmd/go/internal/help/

Reply:
you can set GOCACHE to /tmp which is writable by everyone

Reply:
Yes i just did that, it's working now.